pkg/logql/syntax/parser_test.go

```diff
@@ -17,241 +17,240 @@ func NewStringLabelFilter(s string) *string {
 	return &s
 }
 
-func TestParse(t *testing.T) {
```

Comment:
This is the biggest change so that I can reuse the test cases. Should I split this out into another PR?

Reply:
With biggest I've meant the most line changes because the indention changed. That's about it.

Reply:
Ah, right. I review without whitespace, so that explains the confusion